.github/workflows/docker-build.yml

```diff
@@ -4,29 +4,54 @@ on:
   push:
     branches:
       - master
+  release:
+    types:
+      - published
 
 jobs:
   publish:
     runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v2
+
+      - name: Get Release Tags
+        id: get_release_tags
+        run: |
+          echo "::set-env name=RELEASE_TAG::$(echo ${GITHUB_REF:10})"
+          echo "::set-env name=TAG::stable"
+          echo "::set-env name=TAG_SEMVER::true"
+        if: github.event_name == 'release'
+
+      - name: Get Push Tags
+        id: get_push_tags
+        run: |
+          echo "::set-env name=TAG::latest"
+          echo "::set-env name=RELEASE_TAG::$(echo ${GITHUB_REF:11})"
+          echo "::set-env name=TAG_SEMVER::false"
+        if: github.event_name == 'push'
+
       - name: Publish to docker hub (server)
         uses: elgohr/Publish-Docker-Github-Action@master
         with:
           name: deepforge/server
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          tags: "latest"
+          tags: "${{ env.TAG }},${{ env.RELEASE_TAG }}"
+          tag_semver: "${{ env.TAG_SEMVER }}"
 
       - name: Publish to docker hub (kitchen-sink)
         uses: elgohr/Publish-Docker-Github-Action@master
+
         with:
           name: deepforge/kitchen-sink
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          tags: "latest"
+          tags: "${{ env.TAG }},${{ env.RELEASE_TAG }}"
           dockerfile: Dockerfile.kitchensink
+          buildargs: "${{ env.TAG }}"
```

Comment:
Shouldn't this be `TAG=${{ env.TAG}}`?

Reply:
Yeah. This should be  `TAG=${{ env.TAG }}`